includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -622,9 +622,9 @@ public function get_order_items() {
 				
 				// Set item meta
 				if (function_exists('wc_display_item_meta')) { // WC3.0+
-					$data['meta'] = wc_display_item_meta( $item, array(
+					$data['meta'] = wc_display_item_meta( $item, apply_filters( 'wpo_wcpdf_display_item_meta_args', array(
 						'echo'      => false,
-					) );
+					) ) );
```

Comment:
I would add `$this` (the document object) as a second filter argument, so that people can restrict the changes to specific documents

Reply:
i added it. was it done correctly?

Reply:
👍